.github/workflows/release.yml

```diff
@@ -19,6 +19,7 @@ on:
 
 permissions:
   contents: write
+  id-token: write
```

Comment:
Note that all calling release workflows in the repos need to grant `id-token: write` too.